packages/api/src/routes/middlewares/auth.ts

```diff
@@ -9,49 +11,77 @@ import { getCxIdOrFail } from "../util";
  * Process the API key and get the customer id.
  * The customer id is stored on the Request, property 'cxId'.
  */
-export const processAPIKey = (req: Request, res: Response, next: NextFunction): void => {
+export function processCxId(req: Request, res: Response, next: NextFunction): void {
```

Comment:
nit: why update name to `processCxId` if we are still getting the cxId from the apiKey